actionpack/lib/action_controller/metal/renderers.rb

```diff
@@ -11,28 +11,61 @@ def self.remove_renderer(key)
     Renderers.remove(key)
   end
 
+  # RFC: Where is this used?
   class MissingRenderer < LoadError
```

Comment:
good spleunking.  Now, the code over there is

```ruby
    # This is the common behavior for formats associated with APIs, such as :xml and :json.
    def api_behavior
      raise MissingRenderer.new(format) unless has_renderer?
      # etc
    end

    # Check whether the necessary Renderer is available
    def has_renderer?
      Renderers::RENDERERS.include?(format)
    end
```

which makes me think either the exception should be defined in responders  or `has_renderer?` should be defined here, as well.